draft-ietf-tls-esni.md

```diff
@@ -220,11 +220,16 @@ The ECH configuration is defined by the following `ECHConfig` structure.
         HpkeSymmetricCipherSuite cipher_suites<4..2^16-4>;
     } HpkeKeyConfig;
 
+    struct {
+        ECHConfigExtensionType ech_config_extension_type;
+        opaque extension_data<0..2^16-1>;
```

Comment:
Extremely nitpicky nitpick: I guess it's `ech_config_extension_type` instead of `extension_type` to distinguish from Extension's `extension_type`, but then `extension_data` is still the same. This is a little odd.

I might suggest we just call them `extension_type` and `extension_data` or perhaps even `type` and `data`. There's no actual requirement that field names be unique.

Reply:
Good point -- `type` and `data` is much more succinct!